tests/users.test.js

```diff
@@ -0,0 +1,572 @@
+          Authorization: `Bearer ${token}`,
+        },
+        data: JSON.stringify(user),
+      });
+      expect(result).toEqual(user);
+    });
+  });
+
+  test('Disable should handle a conflict error', () => {
+    const errorResponse = {
+      response: {
+        status: 401,
+      },
+    };
+    axios.request.mockRejectedValueOnce(errorResponse);
+
+    const expectedUrl = `${users_url}/users/${user_id}/disable`;
+
+    const sdk = new mfsdk({ usersUrl: users_url });
+    return sdk.users.Disable(user, token).then(result => {
+      expect(axios.request).toHaveBeenCalledWith({
+        url: expectedUrl,
+        method: 'post',
+        maxBodyLength: 2000,
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+        data: JSON.stringify(user),
+      });
+      console.log(result);
+    });
+  });
+
+  test('Enable should enable user and return success', () => {
+    axios.request.mockResolvedValueOnce({ data: user });
+
+    const expectedUrl = `${users_url}/users/${user_id}/enable`;
+
+    const sdk = new mfsdk({ usersUrl: users_url });
+    return sdk.users.Enable(user, token).then(result => {
+      expect(axios.request).toHaveBeenCalledWith({
+        url: expectedUrl,
+        method: 'post',
+        maxBodyLength: 2000,
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+        data: JSON.stringify(user),
+      });
+      expect(result).toEqual(user);
+    });
+  });
+
+  test('Enable should handle a conflict error', () => {
+    const errorResponse = {
+      response: {
+        status: 401,
+      },
+    };
+    axios.request.mockRejectedValueOnce(errorResponse);
+
+    const expectedUrl = `${users_url}/users/${user_id}/enable`;
+
+    const sdk = new mfsdk({ usersUrl: users_url });
+    return sdk.users.Enable(user, token).then(result => {
+      expect(axios.request).toHaveBeenCalledWith({
+        url: expectedUrl,
+        method: 'post',
+        maxBodyLength: 2000,
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${token}`,
+        },
+        data: JSON.stringify(user),
+      });
+      console.log(result);
+    });
+  });
+
+  test('Authorise User should authorise a user and return success', () => {
+    axios.request.mockResolvedValueOnce({ data: true });
+
+    const expectedUrl = `${users_url}/authorize`;
+
+    const sdk = new mfsdk({ usersUrl: users_url });
+    const access_request = {
+      "subject": user_id,
+      "object": group_id,
+      "action": action,
+      "entity_type": entity_type
+    }
+    return sdk.users.AuthoriseUser(user_id, group_id, action, entity_type, token).then(result => {
+      expect(axios.request).toHaveBeenCalledWith({
+        url: expectedUrl,
+        method: 'post',
+        maxBodyLength: 2000,
+        headers: {
+          'Content-Type': 'application/json',
```

Comment:
this would be set by axios to json when passing js onjects so no need to set it and in other parts of code